neurokit2/complexity/__init__.py

```diff
@@ -18,6 +18,7 @@
 # Fractal
 from .fractal_dfa import fractal_dfa
 from .fractal_mandelbrot import fractal_mandelbrot
+from .fractal_dimension import fractal_dimension
```

Comment:
I think we should name these directly `fractal_higushi` and `fractal_katz` otherwise it's a bit confusing coz our `fractal_*` already implies "fractal dimension using ... method"